feature/experimental/qos/ingress_traffic_classification_and_rewrite/README.md

```diff
@@ -0,0 +1,83 @@
+# QoS-1.1: Ingress traffic classification and rewrite
+
+## Summary
+
+This test aims to validate the functionality of ingress traffic classification and subsequent packet remarking (rewrite) on a Device Under Test (DUT). The DUT's configuration will be evaluated against the OpenConfig QOS model, and traffic flows will be analyzed to ensure proper classification, marking, and forwarding.
+
+## Procedure
+
+* DUT Configuration:
+    * Configure the DUT's ingress and egress interfaces.
+    * Apply QoS classifiers using the OpenConfig QOS model, matching packets based on DSCP/TC/EXP values as per the classification table.
+    * Configure packet remarking rules based on the marking table.
+
+* Traffic Generation:
+    * Use the traffic generator to send:
+        * IPv4 packets with various DSCP values
+        * IPv6 packets with various TC values
+        * MPLS packets with different EXP values, ensuring scenarios with and without label popping.
```

Comment:
* MPLS traffic with swap action.
* IPv4-over-MPLS traffic with pop action
* IPv6-over-MPLS traffic with pop action
* IPv4-over-MPLS-over-IPv4GRE with GRE decapsulate action and MPLS swap/forward
* IPv6-over-MPLS-over-IPv4GRE with GRE decapsulate action and MPLS swap/forward
* IPv4-over-MPLS-over-IPv4GRE with GRE decapsulate action and MPLS pop and IPv4 forward
* IPv6-over-MPLS-over-IPv4GRE with GRE decapsulate action and MPLS pop and IPv6 forward

Reply:
We are currently focused on testing traffic classification and rewriting. Introducing GRE/MPLS actions might overcomplicate this specific test, as we have separate tests dedicated to evaluating MPLS swap/pop/forward actions and GRE decapsulation.

Reply:
Seems like the swap and pop should be covered here.  They can be subtests with specific id's specified to help tie together the readme and code.   (separate t.Run's each with a QOS-1.1.x identifier)

It's not a hard rule, but it seems good to add the GRE decap actions to a separate test focused on GRE decap. If anything to separate pass/fail results for this functionality that builds on top of the base MPLS qos features.

Reply:
yes the MPLS actions are covered here by Generating "MPLS packets with different EXP values, ensuring scenarios with and without label popping."
I will change it as sugrimov suggested with : 
* MPLS traffic with swap action.
* IPv4-over-MPLS traffic with pop action.
* IPv6-over-MPLS traffic with pop action.